### ISSUE TITLE
aws_ses_event_destination: Multiple dimensions per CloudWatch destination

### ISSUE THREAD

**Issue by NiteKrawler0** [terraform-providers/terraform-provider-aws#5620]
The `aws_ses_event_destination` does not appear to support multiple dimensions per CloudWatch Destination.

I am able to add multiple dimensions via the AWS console and it also appears that CloudFormation supports this as well: [AWS::SES::ConfigurationSetEventDestination](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ses-configurationseteventdestination.html)

I have tried various methods for adding additional dimensions, but keep getting the following error message (Examples 1 & 2): 
`Error: aws_ses_event_destination.cloudwatch-aw: cloudwatch_destination: attribute supports 1 item maximum, config has 2 declared`.

Other methods just try and apply 1 dimension to the CloudWatch Destination (Example 3)

Any help/suggestions welcome

### Resource Config

*Example 1*
```
resource "aws_ses_event_destination" "cloudwatch-aw" {
  name                   = "ics-aw-delivery"
  configuration_set_name = "${aws_ses_configuration_set.ics-aw-stats.name}"
  enabled                = true
  matching_types         = ["bounce", "delivery", "reject", "send"]

  cloudwatch_destination = [
    {
      default_value  = "aw-1"
      dimension_name = "ses:configuration-set"
      value_source   = "messageTag"
    },
    {
      default_value  = "aw-2"
      dimension_name = "ses:source-ip"
      value_source   = "messageTag"
    }
  ]
}
```

*Example 2*
```
resource "aws_ses_event_destination" "cloudwatch-aw" {
  name                   = "ics-aw-delivery"
  configuration_set_name = "${aws_ses_configuration_set.ics-aw-stats.name}"
  enabled                = true
  matching_types         = ["bounce", "delivery", "reject", "send"]

  cloudwatch_destination = {
    default_value  = "aw-1"
    dimension_name = "ses:configuration-set"
    value_source   = "messageTag"
  }

  cloudwatch_destination = {
    default_value  = "aw-2"
    dimension_name = "ses:source-ip"
    value_source   = "messageTag"
  }
}
```

*Example 3*
```
resource "aws_ses_event_destination" "cloudwatch-aw" {
  name                   = "ics-aw-delivery"
  configuration_set_name = "${aws_ses_configuration_set.ics-aw-stats.name}"
  enabled                = true
  matching_types         = ["bounce", "delivery", "reject", "send"]

  cloudwatch_destination = {
    default_value  = "aw-1"
    dimension_name = "ses:configuration-set"
    value_source   = "messageTag"

    default_value  = "aw-2"
    dimension_name = "ses:source-ip"
    value_source   = "messageTag"
  }
}
```

### Terraform Version

Terraform v0.11.7

**Comment by NiteKrawler0** [comment]
Hi, anybody have any feedback/update on this?

**Comment by david-wells-1** [comment]
I have also come across this issue with v0.11.8 - am going to submit a PR which resolves the issue

**Comment by bflad** [comment]
Support for multiple `cloudwatch_destination` configuration blocks has been merged and will release with version 1.51.0 of the AWS provider, in the next day or two. 👍

**Comment by bflad** [comment]
This has been released in [version 1.51.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1510-december-05-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by CharlieUPB** [comment]
Is this also possible with sns_destination? Like having multiple topics as event destinations.

Example:

```
resource "aws_ses_event_destination" "sns" {
  name                   = "event-destination-sns"
  configuration_set_name = "${aws_ses_configuration_set.example.name}"
  enabled                = true
  matching_types         = ["bounce", "send"]

  sns_destination {
    topic_arn = "${aws_sns_topic.example.arn}"
  }
  sns_destination {
    topic_arn = "${aws_sns_topic2.example.arn}"
  }
  sns_destination {
    topic_arn = "${aws_sns_topic3.example.arn}"
  }
}
```
